src/lib/LUA/lua.cpp

```diff
@@ -49,6 +48,11 @@ static uint8_t luaSelectionOptionMax(const char *strOptions)
   }
 }
 
+uint8_t luaSelectionFindLabel(const void *luaStruct, uint8_t value, uint8_t *outarray)
+{
+  const struct luaItem_selection *p1 = (const struct luaItem_selection *)luaStruct;
+}
+
```

Comment:
This function doesn't do anything and doesn't seem to be used either!

Reply:
my plan is to complete this function, and use it to fill in the dynamic folder name...that is currently just populated with vtxFolderDynamic name (still WIP here)

Reply:
> my plan is to complete this function, and use it to fill in the dynamic folder name...that is currently just populated with vtxFolderDynamic name (still WIP here)

Fair call, I was jumping the gun 😆